react/react1/week1/my-app/src/TodoList.js

```diff
@@ -0,0 +1,11 @@
+import React from "react";
+import {Listitems} from "./Listitems";
+export function TodoList(props){
+    return (
```

Comment:
Again, I prefer not to inline the map operations in the JSX. Assign it to a const before the return and give it a meaningful name

Reply:
as per your suggestion I have made map operation outside the JSX and assigned it to a const. please review the changes and let me know if I can merge my pull request. thank you